crates/solidity/testing/snapshots/bindings_output/yul/loops/generated/0.4.11-success.txt

```diff
@@ -0,0 +1,51 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+    ╭─[input.sol:1:1]
+    │
+  1 │ contract AssemblyLoops {
+    │          ──────┬──────  
+    │                ╰──────── def: 1
+  2 │     function yul_for_loop() public returns (uint256 z) {
+    │              ──────┬─────                           ┬  
+    │                    ╰─────────────────────────────────── def: 2
+    │                                                     │  
+    │                                                     ╰── def: 3
+    │ 
+  4 │             for { let i := 0 } lt(i, 10) { i := add(i, 1) } { z := add(z, 1) }
+    │                       ┬           ┬        ┬        ┬         ┬        ┬  
+    │                       ╰─────────────────────────────────────────────────── def: 4
+    │                                   │        │        │         │        │  
+    │                                   ╰─────────────────────────────────────── ref: 4
+    │                                            │        │         │        │  
+    │                                            ╰────────────────────────────── ref: 4
+    │                                                     │         │        │  
+    │                                                     ╰───────────────────── ref: 4
+    │                                                               │        │  
+    │                                                               ╰─────────── ref: 3
+    │                                                                        │  
+    │                                                                        ╰── ref: 3
+    │ 
+  8 │     function yul_while_loop() public returns (uint256 z) {
+    │              ───────┬──────                           ┬  
+    │                     ╰──────────────────────────────────── def: 5
+    │                                                       │  
+    │                                                       ╰── def: 6
+    │ 
+ 10 │             let i := 0
```

Comment:
unrelated to this PR: is there a way in `ariadne` to increase the "context lines" around these hints?
Rendering the rest of the lines in this snapshot would make them much more readable/useful.

Reply:
I checked this same thing when I implemented the snapshots output and didn't find a way. But yeah, it would be nice to have that.

Reply:
Submitted a feature request here: https://github.com/zesterer/ariadne/issues/127